src/helpers/sdks.js

```diff
@@ -4,18 +4,33 @@ const utils = require('../utils');
 
 module.exports = {
   getAndroidSDKInfo: () => {
+    let ANDROID_HOME = process.env.ANDROID_HOME;
+
+    try {
+      const localConfigurationPath = path.join(process.cwd(), 'android/local.properties');
```

Comment:
I realize this is probably targeted at react native - but what happens in a regular android project?

Reply:
We may "try" a few different paths for this (accounting for a normal android project, or perhaps other hybrid-ish frameworks), then I think we can merge.